src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package task;
+import java.time.LocalDateTime;
+
+/**
+ * This class encapsulates a Deadline.
+ */
+
+public class Deadline extends Task {
+    private String deadlineBy;
+    private LocalDateTime deadlineInDateTime;
```

Comment:
Good naming. It is very intuitive and explains the named entity to the reader accurately and at a sufficient level of detail.